tests/test_response.py

```diff
@@ -46,7 +46,7 @@
     async def get(req, resp):
         resp.status = HTTPStatus.NO_CONTENT
 
-    await client.get('/test')
+    response = await client.get('/test')
```

Comment:
Out of curiosity, does it change anything given that it's not in use?

Reply:
That's a left over of many, many, MANY changes i did with the tests and app callback... It needs thorough cleaning, most probably.